mario-less.c

```diff
@@ -0,0 +1,30 @@
+#include <cs50.h>
+#include <stdio.h>
+
+int main(void)
```

Comment:
This is not the solution for mario less but for mario more.

Also I found the near exact solution online. Did you code this or just copied it?